app/build.gradle

```diff
@@ -123,17 +123,15 @@ custom.eachFile() { file ->
     return
   }
   map.put(fileName, file.getAbsolutePath())
-
-
 }
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion '25.0.0'
+  compileSdkVersion 25
```

Comment:
Its a good idea to keep your compile and target SDK up to date or a new version of Android might break something. I did discuss it with you briefly, sorry for not making it more clear where the change occured.